samples/bluetooth/peripheral_rscs/src/main.c

```diff
@@ -64,15 +84,15 @@ int set_cumulative(uint32_t total_distance)
 	return 0;
 }
 
-int calibrarion(void)
+int calibration(void)
 {
 	printk("Start the calibration procedure\n");
 	return 0;
 }
 
 void update_loc(uint8_t location)
 {
-	printk("Update sensor location: %d\n", location);
+	printk("Update sensor location: %u - %s\n", location, sensor_location[location]);
```

Comment:
What if `location` is larger than size of `sensor_location`?

Reply:
Added the assertion for checking this